lingpy/sequence/ngrams.py

```diff
@@ -0,0 +1,685 @@
+# *-* coding: utf-8 *-*
+"""
+This module provides various methods for generating and
+collecting n-grams from sequences. Standard ngrams,
+skip ngrams, and positional ngrams can be collected.
+"""
+
+from itertools import chain, combinations, product
+from functools import partial
+
+def _seq_as_tuple(sequence):
+    """
+    Internal function for automatically converting a
+    string sequence to a tuple, if needed.
+
+    Parameters
+    ----------
+    sequence: list or str
```

Comment:
Is this 50 characters per column? Looks very small the text, but our PEP version allows 80. It's easier to read if they are all same size.

Reply:
I've moved to 80 columns both the documentation and the comments.